Purchases/Public/RCPurchaserInfo.h

```diff
@@ -34,7 +34,7 @@ NS_SWIFT_NAME(Purchases.PurchaserInfo)
 @property (readonly) NSSet<NSString *> *nonConsumablePurchases;
 
 /**
- Returns the version number for the version of the application when the user bought the app.
```

Comment:
@aboedo I updated it keeping the old pattern (before my PR) and using the human friendly and widely known names for those properties in the first row of the description. `CFBundleVersion` while being the right technical term, it can easily be mistaken for the `app version` and create confusion.

The new description is:
```
Returns the build number (in iOS) or the marketing version (in macOS) for the version of the application when the user bought the app.
This corresponds to the value of CFBundleVersion (in iOS) or CFBundleShortVersionString (in macOS) in the Info.plist file when the purchase was originally made.
Use this for grandfathering users when migrating to subscriptions.
```